docs/scripts_md/delete_mri_upload.md

```diff
@@ -0,0 +1,171 @@
+# NAME
+
+delete\_mri\_upload.pl -- Delete eveything that was produced by the MRI pipeline for a given MRI upload
+
+# SYNOPSIS
+
+perl delete\_mri\_upload.pl \[-profile file\] \[-i\] \[-n\]
+
+Available options are:
+
+\-profile     : name of the config file in `../dicom-archive/.loris_mri` (defaults to `prod`).
+
+\-i           : when performing the file backup, ignore files that do not exist or are not readable
+               (default is to abort if such a file is found). This option is ignored if `-n` is used.
+
+\-n           : do not backup the files produced by the MRI pipeline for this upload (default is to
+               perform a backup).
+
+# DESCRIPTION
+
+This program deletes all the files and database records produced by the MRI pipeline for a given 
+MRI upload. More specifically, the script will remove the records associated to the MRI upload whose
+ID is passed on the command line from the following tables: `notification_spool`, `tarchive_series`
+`tarchive_files`, `files_intermediary`, `parameter_file`, `files`, `mri_violated_scans`
+`mri_violations_log`, `MRICandidateErrors`, `mri_upload` and `tarchive`. It will also delete from 
+the file system the files that are associated to the upload and are listed in tables `files`
```

Comment:
The script deletes the  archive listed in table `tarchive`, which is listed in the list of tables associated to the upload. Please advise on what you want me do to here.

Reply:
The script deletes the records from tables `mri_violations_log`, `mri_violated_scans` and `MRICandidateErrors` but does not remove any files from `trashbin`. I'll modify the script in order to do this.

Reply:
> The script deletes the archive listed in table `tarchive`, which is listed in the list of tables associated to the upload. Please advise on what you want me do to here.

@nicolasbrossard it was not clear when I read the documentation that the tarchive would also be removed so I would just add that to the documentation so that it is clear that the DICOM archive is also backed up and removed from the filesystem and database

Reply:
Ok, updated the script description.